doc/users/interactive.rst

```diff
@@ -67,54 +68,46 @@ loop is properly integrated with the command line (see
 .. highlight:: ipython
 
 ::
-
-   user@machine:~ $ ipython
```

Comment:
I think it is helpful to identify that we are in an IPython session, we should not assume that our readers well recognize the prompt as IPython from the style.

Reply:
We say that it's IPython in the narrative, but I'm fine putting this line back in. I just don't want them getting tripped up on all the details of your particular Ipython install